apps/src/netsim/NetSimTable.js

```diff
@@ -133,11 +172,11 @@ NetSimTable.prototype.readAll = function (callback) {
  */
 NetSimTable.prototype.makeThrottledRefresh_ = function (waitMs) {
   return _.throttle(function () {
-    this.clientApi_.all(function (err, data) {
-      if (err === null) {
-        this.fullCacheUpdate_(data);
-      }
-    }.bind(this));
+    if (this.useIncrementalRefresh_) {
```

Comment:
I believe this logic belongs in the refreshAll function-- the refresh logic should do whatever kind of partial or complete refresh is appropriate for the particular table.  (In this version of the code refreshAll() is called readAll().)